hack/ci/e2e.sh

```diff
@@ -97,18 +97,83 @@ build() {
 
 # up a cluster with kind
 create_cluster() {
+    # create the audit-policy necessary for API Coverage
+    # https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-policy
+    cp $(dirname $0)/audit-policy.yaml /tmp/audit-policy.yaml
+    touch "${ARTIFACTS}/apiserver-audit.log"
     # create the config file
     cat <<EOF > "${ARTIFACTS}/kind-config.yaml"
 # config for 1 control plane node and 2 workers
 # necessary for conformance
 kind: Cluster
 apiVersion: kind.sigs.k8s.io/v1alpha3
 nodes:
-# the control plane node
+# the control plane node / apiservers
 - role: control-plane
+  extraMounts:
+  - hostPath: /tmp/audit-policy.yaml
+    containerPath: /etc/kubernetes/audit-policy.yaml
+  - hostPath: "${ARTIFACTS}/apiserver-audit.log"
+    containerPath: /var/log/apiserver-audit.log
 - role: worker
 - role: worker
 EOF
+    # open to other ways to detect the kubeadm version
+    # MasterConfiguration changed to ClusterConfiguration in 1.13
+    if echo $JOB_NAME | grep 1-11\\\|1-12
```

Comment:
this seems pretty wildly brittle

Reply:
It it, I couldn't find a way to detect the version and wanted to test the logic in the jobs.
Do we have a recommended way to detect the k8s version during the job?